docs/content/docs/reference/react/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Getting started'
+---
+
+## Installation
+
+With Yarn:
+
+```sh
+yarn add @nhost/react
+```
+
+With Npm:
+
+```sh
+npm install @nhost/react
+```
+
+## Configuration
+
+`@nhost/react` exports a React provider `NhostProvider` that makes the authentication state and the several hooks available in your application. Wrap this component around your whole App.
+
+```jsx
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+import { NhostProvider } from '@nhost/react'
+import { Nhost } from '@nhost/client'
+
+import App from './App'
+
+const nhost = new Nhost({
+  backendUrl: 'http://localhost:1337'
+})
+
+ReactDOM.render(
+  <React.StrictMode>
+    <NhostProvider nhost={nhost}>
+      <App />
+    </NhostProvider>
+  </React.StrictMode>,
+  document.getElementById('root')
+)
+```
+
+### Options
+
+```js
+const nhost = new Nhost({ backendUrl, autoLogin, autoRefreshToken, storageGetter, storageSetter })
+```
+
+| Name               | Type                                | Default          | Notes                                                                                                                                                                                                                                          |
+| ------------------ | ----------------------------------- | ---------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `backendUrl`       | string                              |                  | The Nhost app url, for instance `https://my-app.nhost.run`. When using the CLI, its value is `http://localhost:1337`                                                                                                                           |
+| `autoLogin`        | boolean                             | `true`           | If set to `true`, the client will detect credentials in the current URL that could have been sent during an email verification or an Oauth authentication. It will also automatically authenticate all the active tabs in the current browser. |
```

Comment:
`autoLogin` should be `autoSignIn` if we want to be consistent. Which I think we should.

Reply:
I used the same name as in nhost-js and hasura-auth-js. Changed to autoSignIn